first-blog-post.md

```diff
@@ -0,0 +1,28 @@
+# Welcome to OctoBot blog
+
+Since 2018, OctoBot is an open source cryptocurrency trading bot that allow hundred of users to trade easily with professional features.
```

Comment:
```suggestion
Since 2018, OctoBot is an open source cryptocurrency trading bot that allow hundreds of users to trade easily with professional features.
```